Misc/NEWS.d/next/Core and Builtins/2022-02-10-02-29-12.bpo-46323.HK_cs0.rst

```diff
@@ -0,0 +1,3 @@
+:mod:`ctypes` now allocates memory on the stack instead of on the heap
```

Comment:
There was no heap allocation directly before this PR, and there will be no heap allocation after, right? It's just eliminating that dead code path.

Reply:
@sweeneyde Yeah but this is the result of bpo-45323

Reply:
Victor and I decided to add NEWS entry for all this change at this moment.
Theoretically, we had to add NEWS entry at the first PR, but it's too late.